agent/functional_tests/testdata/taskdefinitions/oom-container/task-definition.json

```diff
@@ -5,8 +5,8 @@
     "memory": 64,
     "name": "error",
     "cpu": 1,
-    "image": "127.0.0.1:51670/python:2",
-    "command": ["python", "-c", "foo=' '*1024*1024*512; import time; time.sleep(10)"]
+    "image": "amazon/amazon-ecs-stress:make",
+    "command": ["-concurrency", "10", "-memory", "100"]
```

Comment:
Have you run the test multiple times to see whether all tests passed? I think feng made some changes to increase the memory we allocated here in order to make the test less flaky. You may wanna to double check the new changes will not break it.